draft-jhoyla-req-mtls-flag.md

```diff
@@ -114,15 +143,27 @@ terminate the connection with a fatal `illegal_parameter` alert.
 
 # Security Considerations
 
-This flag should have no effect on the security of TLS, as the server may
-always send a `CertificateRequest` message during the handshake. This flag
-merely provides a hint that the client will handle the request gracefully.
-Because the server acknowledges the flag in the `CertificateRequest` the client
-can always be sure whether a `CertificateRequest` was triggered by
-`request_client_auth` or not.
-
-TODO: Discuss security considerations related to a flag that changes which
-trust anchors are offered and how to handle/authorise application data.
+On its own, this flag should have no effect on the security of TLS, as the
+server may always send a `CertificateRequest` message during the handshake.
+This flag merely provides a hint that the client will handle the request
+gracefully. Because the server acknowledges the flag in the
+`CertificateRequest` the client can always be sure whether a
+`CertificateRequest` was triggered by `request_client_auth` or not.
+
+However, the use case for which the flag is enabled may entail significant
+changes to how the client selects a certificate and what trust anchors the
+server uses to validate it. The use case may also impact how the server handles
+application data transmitted over the established connection.
+
+In the case of identifying web crawlers (see {{introduction}}), a web server
+would largely handle traffic from a web crawler as if it were any
+unauthenticated client. If for instance the server already requests a
+certificate to grant certain clients access to some resource, the server must take
+care to ensure this access is not also granted to the web crawler.
+
+When the server accepts the `request_client_auth` flag, it SHOULD verify the
+certificate against a set of trust anchors that is disjoint from the set of
+trust anchors it uses when the flag is ignored or rejected.
```

Comment:
not sure this is the right action. my understanding is if a server accepts the flag but does not verify the certificate, there's a high change of misimplementation, and we should recommend to terminate the connection

```suggestion
trust anchors it uses when the flag is ignored or rejected. If it does
not, it is RECOMMENDED to terminate the connection.
```